lib/gemdiff/repo_finder.rb

```diff
@@ -101,7 +101,7 @@ def gemspec_homepage(gem_name)
         return nil unless (yaml = gemspec(gem_name))
         spec = YAML.load(yaml)
         return secure_url(spec.homepage) if spec.homepage =~ GITHUB_REPO_REGEX
-        match = spec.description.match(GITHUB_REPO_REGEX)
+        match = spec.description.match(GITHUB_REPO_REGEX) if spec.description
```

Comment:
Or perhaps `spec.description.to_s.match(GITHUB_REPO_REGEX)`?

Reply:
@bquorning Agree. Changed in 61c1ef0